docs/getting-started/application-setup/07-further-reading.md

```diff
@@ -0,0 +1,12 @@
+---
+pagination_next: null
+---
```

Comment:
Is there a reason why this doesn't continue forward to the *Development* section after *Application Setup*? I would expect it to allow the reader to continue, but can see that we're ending a section here as well. I don't think this needs to be changed, but I wanted to call it out. The `pagination_next:` property is great and I appreciate the usage of it in this PR.

Reply:
That's a great question. My initial thought was that since the application setup is complete at that point and it already lists several options the reader could go to next (Development, Frontend Guides, Backend Guides, etc), it seemed like a natural "concluding" point for the section. But now that I think about it, I could totally see how readers would expect the continue to the next section... I'll update it to go to the next section 😄